package.json

```diff
@@ -65,21 +65,21 @@
     "sentry:publish": "node ./development/sentry-publish.js",
     "lint": "yarn lint:prettier && yarn lint:eslint && yarn lint:tsc && yarn lint:styles",
     "lint:fix": "yarn lint:prettier:fix && yarn lint:eslint:fix && yarn lint:styles:fix",
-    "lint:prettier": "prettier --check -- '**/*.json'",
-    "lint:prettier:fix": "prettier --write -- '**/*.json'",
-    "lint:changed": "./development/get-changed-file-names.sh | grep -E --regexp='[.](js|ts|tsx)$' | tr '\\n' '\\0' | xargs -0 eslint",
-    "lint:changed:fix": "./development/get-changed-file-names.sh | grep -E --regexp='[.](js|ts|tsx)$' | tr '\\n' '\\0' | xargs -0 eslint --fix",
+    "lint:prettier": "prettier --check --cache -- '**/*.json'",
+    "lint:prettier:fix": "prettier --write --cache -- '**/*.json'",
+    "lint:changed": "./development/get-changed-file-names.sh | grep -E --regexp='[.](js|ts|tsx)$' | tr '\\n' '\\0' | xargs -0 eslint --cache --cache-location node_modules/.cache/eslint/.eslint-cache",
+    "lint:changed:fix": "./development/get-changed-file-names.sh | grep -E --regexp='[.](js|ts|tsx)$' | tr '\\n' '\\0' | xargs -0 eslint --fix --cache --cache-location node_modules/.cache/eslint/.eslint-cache",
     "lint:changelog": "auto-changelog validate",
     "lint:changelog:rc": "auto-changelog validate --rc",
-    "lint:eslint": "eslint . --ext js,ts,tsx,snap --cache",
-    "lint:eslint:fix": "yarn lint:eslint --fix",
+    "lint:eslint": "eslint . --ext js,ts,tsx,snap --cache --cache-location node_modules/.cache/eslint/.eslint-cache",
+    "lint:eslint:fix": "yarn lint:eslint --fix --cache --cache-location node_modules/.cache/eslint/.eslint-cache",
```

Comment:
Since I was in here doing a bunch of lint work I added caching. Let me know if you want all these cache changes in a separate PR.